src/include/processor/operator/sink.h

```diff
@@ -44,5 +44,24 @@ class Sink : public PhysicalOperator {
     std::unique_ptr<ResultSetDescriptor> resultSetDescriptor;
 };
 
+class DummySink : public Sink {
```

Comment:
Hmm, I wasn't expecting that we need a DummySink operator. We have built prior subplans that build semi-masking pipelines for other parts of the query plan, did we not? If so, what was the Sink operator in those subplans?

Reply:
As we discussed, this is needed.